examples/diffusion/circle.py

```diff
@@ -41,7 +41,7 @@
 Using this mesh, we can construct a solution variable
 
 .. index::
-   object: fipy.variables.cellVariable.CellVariable
+   pair: object; fipy.variables.cellVariable.CellVariable
```

Comment:
What is this supposed to do? When I go to the [documentation](https://pages.nist.gov/fipy/en/tweak_documentation/generated/examples.diffusion.circle.html#module-examples.diffusion.circle), I'm not seeing anything to click on or anything at that point in the documentation.